src/main/java/com/digirati/taxonomy/manager/normalisation/TextNormaliser.java

```diff
@@ -0,0 +1,80 @@
+package com.digirati.taxonomy.manager.normalisation;
+
+import edu.stanford.nlp.ling.CoreLabel;
+import edu.stanford.nlp.pipeline.CoreDocument;
+import edu.stanford.nlp.pipeline.StanfordCoreNLP;
+
+import java.util.Properties;
+import java.util.Set;
+
+/**
+ * Provides a mechanism for normalising any text inputs, including the terms to search for and the
+ * text in which to search.
+ */
+public class TextNormaliser {
+
+    private final Set<String> stopwords;
+
+    private StanfordCoreNLP pipeline;
+
+    /**
+     * Initialises a {@link TextNormaliser} for normalising any input language. Currently, full
+     * lemmatisation is only supported for English due to a restriction of the NLP library that
+     * we're using, but language-specific tokenisation should still work
+     *
+     * @param stopwords a collection of common boilerplate words to remove from the text (e.g.
+     *     "the", "a", etc.)
+     * @param languageKey the ISO 639-1 language code for the text to be normalised
+     * @param languageName the name of the language of the text to be normalised
+     */
+    public static TextNormaliser initialiseNormaliser(
```

Comment:
This isn't really much different from running this in the constructor. If we return a `Future<TextNormaliser>` and run it on a single threaded `Executor` it'll be much more clear that `new StanfordCoreNLP()` is an expensive call.

Reply:
Really, we just want a way to say "this is a long, blocking operation" -- hiding the initialization behind concurrent APIs to do that is fine, I think.